### DISCUSSION TITLE
Pull requests from other repository

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2366] [Q&A] [answered]
I believe my Adwaita and Adwaita-Dark <a href=https://github.com/AzumaHazuki/lxqt-themes-adwaita>themes</a> are in a consistent enough state for inclusion into mainline, and would like to make a draft pull request. However, because these are in a separate repo rather than a fork or branch of the main lxqt-themes, I am not sure how to do this.

Is it possible to make a PR across repository boundaries? If so, how is this done?

**Comment by shlyakpavel** [comment, marked as answer]
I would fork the repo, add themes in there and open a Pull Request. Although there are [advanced solutions ](https://stackoverflow.com/questions/1683531/how-to-import-existing-git-repository-into-another) to the problem, I doubt they're worth the effort.
